app/assets/stylesheets/components/_supplier-table.scss

```diff
@@ -1,8 +1,10 @@
 .supplier-table {
   margin: $cads-spacing-6 0 $cads-spacing-3 0;
+  width: 768px;
```

Comment:
I think if implementing the workaround noted in this comment: https://github.com/citizensadvice/energy-comparison-table/pull/320/files#r1620318371 will make it possible to remove the set width of the table.